server/plugin/api.go

```diff
@@ -178,6 +185,54 @@ func (p *Plugin) handleError(w http.ResponseWriter, r *http.Request, error *seri
 	}
 }
 
+// handleUnlinkProject unlinks a project
+func (p *Plugin) handleUnlinkProject(w http.ResponseWriter, r *http.Request) {
+	mattermostUserID := r.Header.Get(constants.HeaderMattermostUserIDAPI)
+
+	var project *serializers.ProjectDetails
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&project); err != nil {
+		p.API.LogError(constants.ErrorDecodingBody, "Error", err.Error())
+		p.handleError(w, r, &serializers.Error{Code: http.StatusBadRequest, Message: err.Error()})
+		return
+	}
+
+	projectList, err := p.Store.GetAllProjects(mattermostUserID)
+	if err != nil {
+		p.API.LogError(constants.ErrorFetchProjectList, "Error", err.Error())
+		p.handleError(w, r, &serializers.Error{Code: http.StatusInternalServerError, Message: err.Error()})
+		return
+	}
+
+	if !p.IsProjectLinked(projectList, *project) {
+		p.API.LogError(constants.ProjectNotFound, "Error")
```

Comment:
```suggestion
		p.API.LogError(constants.ProjectNotFound, "Project", project.Name)
```
You can use something other than `Name` as well.